File: sitekit_test.go
package seedaudioaisitekit

import "testing"

func TestSiteMetadata(t *testing.T) {
	site := Site()
	if site.Name != "Seed Audio AI" {
		t.Fatalf("unexpected name: %s", site.Name)
	}
	if site.Homepage != Homepage {
		t.Fatalf("unexpected homepage: %s", site.Homepage)
	}
	if site.CanonicalPages["textToSpeech"] != "https://seedaud.io/text-to-speech/" {
		t.Fatalf("missing text-to-speech page")
	}
}

func TestURL(t *testing.T) {
	if URL("") != "https://seedaud.io/" {
		t.Fatalf("home URL malformed")
	}
	if URL("/pricing/") != "https://seedaud.io/pricing/" {
		t.Fatalf("path URL malformed")
	}
}

File: php/tests/SiteKitTest.php
<?php

require_once __DIR__ . '/../src/SiteKit.php';

use SeedAudioAI\SiteKit\SiteKit;

assert(SiteKit::site()['name'] === 'Seed Audio AI');
assert(SiteKit::site()['homepage'] === 'https://seedaud.io/');
assert(SiteKit::url() === 'https://seedaud.io/');
assert(SiteKit::url('/pricing/') === 'https://seedaud.io/pricing/');
